suite-common/graph/src/balanceHistoryUtils.ts

```diff
@@ -320,6 +322,8 @@ export const getAccountHistoryMovementFromTransactions = ({
         case 'xrp':
             return getAccountHistoryMovementItemRipple({ transactions, from, to });
         case 'eth':
+        case 'pol':
+        case 'bnb':
             return getAccountHistoryMovementItemETH({ transactions, from, to });
```

Comment:
Maybe add a comment to the `getAccountHistoryMovementItemETH` method that it's not only for Ethereum, but for all ethereum-type networks?

BTW, @Nodonisko, can we use `getAccountHistoryMovementItemBTC` for all btc-like coins? Does it make sense or it's better to keep it for BTC only?

Reply:
updated

Reply:
updated

Reply:
It's actually unused code right now, I kept it there because I thinked we will download all transactions anyway, but since we are not doing that is much better and faster to use blockbook to get account movements